### PR TITLE
Switched logging to `pretty` and `standard`.

### PR DESCRIPTION
**Pull request by kshakir**
DSDEEPB-2271

### CODE REVIEW

**Comment by kshakir** [comment]
Choosing @geoffjentry as first reviewer

File: README.md [hunk 1]
```diff
@@ -1210,23 +1210,9 @@ Defaults to "false".
...
-├── cromwell.2015-10-26.log
-├── workflow.319df202-a60f-47c8-b886-bd4821747c68.log
-├── workflow.36e07688-9e47-45bd-9930-aff58471541e.log
-└── workflow.7dad065d-9d7a-4450-91c8-1f7ece184851.log
-```
-
-There would also be logging to the standard out stream as well.
-
-The `cromwell.<date>.log` file contains an aggregate of every log message, while the `workflow.<uuid>.log` files contain only log messages that pertain to that particular workflow.
+* `LOG_ROOT` - Specifies the directory where logs will be written (default `.`). Currently unused, as logs are only written to standard out, but will be restored in a future update.
```
Comment (geoffjentry) [inline README.md]: Petty but I'd put LOG_ROOT in the third position

**Comment by geoffjentry** [comment]
:+1: the wheel has chosen @mcovarr

**Comment by geoffjentry** [comment]
@kshakir Can you make 'standard' the default for server mode? Otherwise looks good

**Comment by kshakir** [comment]
Ok. I'll revert [this](https://github.com/broadinstitute/cromwell/commit/2e3f45bbedeaa4c522751e9ff6f5594c57b88b35#diff-facc2160a82442932c41026c9a1e4b2bL28) change in behavior from a while ago, and update the code to reset logging type based on standard command line arguments. This goes against what the docs currently say, so I'll update those too.

**Comment by mcovarr** [comment]
:+1: seconding the request for standard being the default in server mode

**Comment by kshakir** [comment]
Ready for re-review. Going to leave the commits separate though, as the implementation of server logging config feels sketchy.

File: src/main/scala/cromwell/Main.scala [hunk 2]
```diff
@@ -43,6 +45,22 @@ object Main extends App {
...
+    * Overwrites LOG_MODE system property used in our logback.xml, _before_ the logback classes load.
+    * Restored from similar functionality in
+    *   https://github.com/broadinstitute/cromwell/commit/2e3f45b#diff-facc2160a82442932c41026c9a1e4b2bL28
+    * TODO: Logback is configurable programmatically. We don't have to overwrite system properties like this.
+    *
+    * @param args The command line arguments.
+    */
+  private def setupServerLogging(args: Array[String]): Unit = {
+    args.headOption.map(_.capitalize) match {
+      case Some("Server") => sys.props.getOrElseUpdate("LOG_MODE", "STANDARD")
```
Comment (scottfrazer) [inline src/main/scala/cromwell/Main.scala]: You're calling `_.capitalize` but this case matches `Some("Server")`?

Comment (scottfrazer) [inline src/main/scala/cromwell/Main.scala]: Oh, nevermind...

File: src/main/scala/cromwell/Main.scala [hunk 3]
```diff
@@ -43,6 +45,22 @@ object Main extends App {
    * array becoming null in "new Main(args)" when used with: sbt 'run run ...'
    */
   new Main().runAction(args)
+
+  /**
+    * If a cromwell server is going to be run, makes adjustments to the default logback configuration.
+    * Overwrites LOG_MODE system property used in our logback.xml, _before_ the logback classes load.
+    * Restored from similar functionality in
+    *   https://github.com/broadinstitute/cromwell/commit/2e3f45b#diff-facc2160a82442932c41026c9a1e4b2bL28
+    * TODO: Logback is configurable programmatically. We don't have to overwrite system properties like this.
```
Comment (scottfrazer) [inline src/main/scala/cromwell/Main.scala]: Basically saying "ditch the logback.xml at some point and do it in straight Scala"?

Comment (kshakir) [inline src/main/scala/cromwell/Main.scala]: Someday, yes. Setting system properties is technically possible, but feels strange. Also, I'm not sure the exact details at the moment, but cromwell seems to exists as both a library and an executable. There should be some default logging at various levels for the cromwell library, but maybe only default logging appenders would be setup within the executable.

Comment (geoffjentry) [inline src/main/scala/cromwell/Main.scala]: re existing as a library, the library portion of Cromwell will be moving elsewhere in the near future, c.f. DSDEEPB-2301

**Comment by kshakir** [comment]
Any other comments before merging? Please let me know before 4pm, or it'll wait for another ticket

**Comment by geoffjentry** [comment]
@kshakir If it's not already done and it's not hard can you incorporate DSDEEPB-2339 (and close it if you do)

**Comment by geoffjentry** [comment]
Also it might be good to merge this after call caching, just in case it somehow conflicts

**Comment by kshakir** [comment]
Cool, will put in a bit of logging for DSDEEPB-2239, and wait for #287 to merge first.

**Comment by geoffjentry** [comment]
@kshakir Looks like this just needs some conflict resolution & then can be merged, is that correct? If that's the case and you're not going to be around I can take care of that, just let me know

**Pull request merged**